src/Template/Element/sentence_comments/comment.ctp

```diff
@@ -110,9 +112,13 @@ if ($sentenceOwnerLink) {
             <?= $this->Members->image($username, $avatar, array('class' => 'md-user-avatar')); ?>
         </md-card-avatar>
         <md-card-header-text>
+        <?php if ($username): ?>
             <span class="md-title">
                 <a href="<?= $userProfileUrl ?>"><?= $username ?></a>
             </span>
+        <?php else: ?>
+            <i><?= __('Former member') ?></i>
```

Comment:
```suggestion
            <i><?= h(__('Former member')) ?></i>
```

Reply:
Do you suggest to add `h()` to every translatable string in general?

Reply:
I think we should do so. Of course it’s not always required as sometimes the function you give the translatable string to escapes it for you (such as methods in HtmlHelper). Until we go through all the strings, I try to escape new ones at least.